fcrepo-kernel/src/main/java/org/fcrepo/kernel/exception/ManagedTypeException.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.exception;
+
+/**
+ * Certain mixin types are managed by the repository only.
+ *
+ * @author whikloj
+ * @since 2015-06-02
+ */
+public class ManagedTypeException extends ConstraintViolationException {
```

Comment:
This was the part of this ticket I said I was not sure of in [the comments of the JIRA ticket](https://jira.duraspace.org/browse/FCREPO-1418?focusedCommentId=42814&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-42814). 
This was being disregarded previously [here](https://github.com/fcrepo4/fcrepo4/pull/808/files#diff-ef4f510e2f7370114164c1907486eb9dR149), and I am not entirely sure why or if this is needed at all.

That being said I have no problem removing this and using ServerManagedPropertyException in its place.